.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: go
+
+os:
+  - linux
+  - osx
+
```

Comment:
Could you also add codecov as it done over here https://github.com/multiformats/go-multihash/blob/master/.travis.yml#L14-L19

Thanks